proto/tendermint/mempool/types.proto

```diff
@@ -34,6 +34,6 @@ service MempoolService {
     returns (GetNextTransactionResponse) {}
 }
 
-message MempoolClient {
-  string mem_addres = 1;
+message LocalClient {
+  uint64 mem_address = 1;
```

Comment:
since this goes against golang's memory safety model and type system, please explain in detail how this works and in what condition this could break